web/packages/teleport/src/Integrations/DeleteAwsOidcIntegrationDialog.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Alert, ButtonSecondary, ButtonWarning, P1, Text } from 'design';
+import Dialog, {
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from 'design/DialogConfirmation';
+import useAttempt from 'shared/hooks/useAttemptNext';
+
+import { IntegrationAwsOidc } from 'teleport/services/integrations';
+import useStickyClusterId from 'teleport/useStickyClusterId';
+
+import { type DeleteRequestOptions } from './Operations/IntegrationOperations';
+
+type Props = {
+  close(): void;
+  remove(opt?: DeleteRequestOptions): Promise<void>;
+  integration: IntegrationAwsOidc;
+};
+
+export function DeleteAwsOidcIntegrationDialog(props: Props) {
+  const { close, remove, integration } = props;
+  const { attempt, run } = useAttempt();
+  const isDisabled = attempt.status === 'processing';
+  const { clusterId } = useStickyClusterId();
+
+  const awsResourceExplorerUrl =
+    `https://resource-explorer.console.aws.amazon.com/resource-explorer/home#/search?query=` +
+    `tag%3Ateleport.dev%2Forigin%3Dintegration_awsoidc+` + // tag:teleport.dev/origin=integration_awsoidc
+    `tag%3Ateleport.dev%2Fcluster%3D${clusterId}+` + // tag:teleport.dev/cluster={cluster_name}
+    `tag%3Ateleport.dev%2Fintegration%3D${integration.name}`; // tag:teleport.dev/integration={integration_name}
+  function onOk() {
+    run(() => remove({ deleteAssociatedResources: true }));
+  }
+
+  return (
+    <Dialog onClose={close} open={true}>
+      <DialogHeader>
+        <DialogTitle>Delete Integration?</DialogTitle>
+      </DialogHeader>
+      <DialogContent width="600px">
+        {attempt.status === 'failed' && <Alert children={attempt.statusText} />}
+        <Alert
+          kind="warning"
+          primaryAction={{
+            content: 'AWS Resource Explorer',
+            href: awsResourceExplorerUrl,
+          }}
+        >
+          There may be AWS resources created by this integration that require
+          manual clean up. Visit the AWS Resource Explorer to see resources with
+          tags matching this integration.
+        </Alert>
+        <Alert kind="info">
+          Teleport resources used for auto-discovery that reference this
+          integration will also be removed.
```

Comment:
not a blocker, but i'd consult with UX folks and get consistent design. for `git server` this similar message "Teleport will delete" is inside a `warning` banner instead and is at the bottom of the dialogue.

<img width="564" height="333" alt="Image" src="https://github.com/user-attachments/assets/cf6337ef-a69e-4961-bc81-ea431380d8a1" />